### DISCUSSION TITLE
There are no user parameters. How do I edit an existing bin created with the add on?

### DISCUSSION THREAD

**Discussion by simspace-dev** [Le0Michine/FusionGridfinityGenerator#67] [General]
Love the F360 Gridfinity Generator, great job!

But how do I edit an existing bin created with the add on? There are no user parameters.

The F360 add-on page say's "_The created bodies are fully parametric, empowering you to make precise adjustments and modifications to meet your exact specifications. From dimensions to hole placement, GridfinityGenerator enables you to achieve the perfect fit._"

However, after I create a bin and I open the parameters dialog to update the design, I don't see any user parameters. The only parameters found are under the model parameters where I have to edit a specific feature. 

And if I open the generator it creates a new design, it does not allow me to edit the existing design.

Thanks!

**Comment by JanLJensen** [comment]
I had the same question.  Tried many different ways...no luck.  Oh, and thank you for developing this add-in.  Awesome!

**Comment by booswig** [comment]
Exact same problem. Once the gridfinity bin and baseplate are created, how are you able to go back and change the number of rows and columns if you want it to be different than what you started with. That is, "how do you edit the columns, rows and height"?

I can see users on youtube that are able to edit the columns, rows and height under 'user parameters'. Why are they missing from user parameters?

**Comment by gregreid** [comment]
I found this discussion as I was about to ask the same...

I created a 2x3x3 solid box, spent some hours meticulously planning and building a proper arrangment for my unique tool ... and then realized that I'll need to bump up to a 4-high box for proper grid base clearance.  This ought to be an easy thing to do ... but how?!?!

I"m sure there's an extrude in there somewhere in the parametric timeline that sets the body height, and I'll keep looking for it.  But this really ought to be exposed as a User Parameter, if possible.

Oh, and of course THIANK YOU for making this excellent plug-in.  I've been using it a TON this past couple of weeks, and this is the first time I've found a small gripe with it. :-/
